app/scripts/controllers/statefulSets.js

```diff
@@ -34,6 +34,14 @@ angular.module('openshiftConsole')
           updateFilterWarning();
         }));
 
+        // TODO: 1.6 eliminate this block, we dont actually need pods on this page,
+        // we are just using to fix the fact that the replicas count in inaccurate
+        watches.push(DataService.watch('pods', context, function(podData) {
+          $scope.pods = podData.by('metadata.name');
+          var allOwners = _.toArray($scope.statefulSets);
```

Comment:
i actually dont think you need to convert it to an array, i think it will accept $scope.statefulSets directly.  The only reason we did it on overview is because we have to combine a bunch of different sets of owners